src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+- Round removal should be discouraged where possible. It should not be the optimal choice to gib or leave a player to rot unless in specific circumstances.
+- Revolutionaries should not be discovered through *no fault of their own* in the Initial Conversions phase (i.e. metagameable mechanics). 
+- A single non-Security crew member discovering an antag should not spell the end of the antag faction's progression in the round.
+- Round end conditions should be clear and straight-forward, though a crew overcoming the antag shouldn't necessarily mean the end of the round.
+- Crew shouldn't be able to hard counterplay the antag without engaging with the antag. 
+
+
+## New Revolutionaries, Basic layout
+<sup>Any values given here are suggestions and are especially marked by being **bolded**. Any such value should be adjusted based on playtesting.</sup>
+
+The Revolutionary antag begins either as a 2-4 roundstart antag roll for the Revolutionary gamemode, or a single crewmember secretly becomes a Revolutionary antag during another gamemode midround.
+
+Revolutionary antags that start out this way are considered Head Revolutionaries (HeadRevs). A Head Revolutionary has the duty to begin the conversion of other crew and have some extra tools at their disposal to strengthen the progression of the antag faction, but they are not strictly necessary for the revolution antags to succeed. The goal for Head Revolutionaries is to convert crew to grow the faction's strength, convert department Heads and survive until the evacuation shuttle without being deconverted. 
+
+Head Revolutionaries share the same team and when they are given the role they obtain the names and jobs of the other Head Revolutionaries on the station. Revolutionaries can be identified by other revolutionaries via an icon next to their character, however Head Revs do not have a unique icon.
+
+For crew, the goal is to deconvert revolutionary converts, protect Heads from being converted, and ensure Head Revs are discovered and/or do not make it onto the evacuation shuttle. 
+
+### Win Conditions
+
+Win conditions for a round with a Revolutionary antag are the following:
+
+- Revolutionary Major Victory: No *alive, unrestrained* Command *on the evac shuttle* are non-rev, >**50%** of Command are converted Revolutionaries.
+- Revolutionary Minor Victory:  No *alive, unrestrained* Command *on the evac shuttle* are non-rev, <**50%** of Command are converted Revolutionaries.
+- Rev-Crew Tie: Both Rev Minor & Crew Minor criteria are met, or neither Rev Minor & Crew Minor criteria are met.
+- Crew Minor Victory: No *alive, unrestrained* headrevs are *on the evac shuttle* when Evac reaches CentComm.
+- Crew Major Victory: *All* revolutionaries are *deconverted* or *dead* when Evac reaches CentComm.
```

Comment:
If you just need an alive HR and all Heads to be dead when you reach Centcomm, then this means the HRs can just bide their time and bomb the shit out of the evac/end-of-shift shuttle and completely ignore the conversion mechanic.

Reply:
With what?

If you get to the point where a rev simply waits for evac, not only will they be fighting all of Sec, but they have to deal with actually killing all heads. Realistically this would need to be something unique like a chemist with CLF3 bottles, and there are pushes for making Evac bombing less lethal.

I don't entirely disagree with this point, but it is pretty much the same issue as the Traitors gamemode, and solutions can be applicable to both.